lib/rubocop/cop/bundler/gem_comment.rb

```diff
@@ -88,7 +88,7 @@ class GemComment < Base
         CHECKED_OPTIONS_CONFIG = 'OnlyFor'
         VERSION_SPECIFIERS_OPTION = 'version_specifiers'
         RESTRICTIVE_VERSION_SPECIFIERS_OPTION = 'restrictive_version_specifiers'
-        RESTRICTIVE_VERSION_PATTERN = /<|~>/.freeze
+        RESTRICTIVE_VERSION_PATTERN = /^\s*(?:<|~>|\d|=)/.freeze
```

Comment:
`\A` may be preferable if multi-line is not expected.

```suggestion
        RESTRICTIVE_VERSION_PATTERN = /\A\s*(?:<|~>|\d|=)/.freeze
```

Reply:
Suggestion accepted and branch rebased, thanks!